obspy/core/inventory/response.py

```diff
@@ -1663,6 +1663,56 @@ def get_sacpz(self):
         paz = self.get_paz()
         return paz_to_sacpz_string(paz, self.instrument_sensitivity)
 
+    @staticmethod
+    def from_paz(zeros, poles, stage_sequence_number=1, stage_gain=1,
```

Comment:
The gain should really be a required parameter.

Reply:
And the stage sequence number should IMHO not as this will return a complete response object and you already know that it will be stage 1.